react_excercise/got-app/src/Search.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { Input, InputGroup, Button } from 'reactstrap';
+const { searchForCharacters } = require('./Api');
+
+const Search = () => {
+    const [searchCharacter, setSearchCharacter] = useState([]);
+    const [inputCharacter, setInputCharacter] = useState('');
+    const [error, setError] = useState(null);
+
+    const handleInput = (event) => {
+        setInputCharacter(event.target.value);
```

Comment:
After you search, if there are no results and you try to type again, 'No characters found' remains on the screen. Not a major concern but just not good user experience.

Reply:
fixed.  Reset the error in handleInput once user starts inputting new characters